packages/selection/src/SelectionContainer.spec.tsx

```diff
@@ -166,6 +166,16 @@ describe('SelectionContainer', () => {
         });
       });
 
+      describe('CTRL+ENTER keyCode', () => {
+        it('is not consumed', () => {
+          const { getByText } = render(<BasicExample />);
+          const item = getByText('Item-1');
+
+          fireEvent.keyDown(item, { keyCode: KEY_CODES.ENTER, ctrlKey: true });
```

Comment:
This file is a bit outdated and the project is moving towards the modern usages of `testing-library`. What do you think about trying out the [`userEvent.keyboard`](https://testing-library.com/docs/ecosystem-user-event/#keyboardtext-options) API? It'll better simulate an actual user experience. In this scenario, the item would need focus first, so would require a `userEvent.tab()`.

Feel free to leave as is though.

Reply:
I think things like that should be updated consistently across the file, so let's do it in one go. But I agree that moving to testing-library is a great thing to do.

Reply:
@sunesimonsen feel free to strike through non-applicable PR checklist items, as seen here: https://github.com/zendeskgarden/react-containers/pull/342